multi-node/aws/pkg/cluster/cluster_test.go

```diff
@@ -1,24 +1,40 @@
 package cluster
 
 import (
+	"errors"
 	"testing"
 
 	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/coreos/coreos-kubernetes/multi-node/aws/pkg/config"
 )
 
-type dummyEC2Service map[string]struct {
+const minimalConfigYaml = `
+externalDNSName: test-external-dns-name
+keyName: test-key-name
+region: us-west-1
+availabilityZone: us-west-1c
+clusterName: test-cluster-name
+kmsKeyArn: "arn:aws:kms:us-west-1:xxxxxxxxx:key/xxxxxxxxxxxxxxxxxxx"
+`
+
+type VPC struct {
 	cidr        string
 	subnetCidrs []string
 }
 
+type dummyEC2Service struct {
+	VPCs     map[string]VPC
+	KeyPairs map[string]bool
+}
+
 func (svc dummyEC2Service) DescribeVpcs(input *ec2.DescribeVpcsInput) (*ec2.DescribeVpcsOutput, error) {
 	output := ec2.DescribeVpcsOutput{}
-	for _, vpcId := range input.VpcIds {
-		if vpc, ok := svc[*vpcId]; ok {
+	for _, vpcID := range input.VpcIds {
```

Comment:
This capitalization makes golint happy, but it's inconsistent with the amazon's naming.  I'd prefer to not propagate their deviancy, but I can change this back if we'd rather be consistent.